src/internal/model_render.go

```
@@ -253,71 +253,101 @@
-		renderIndex++
+func computeWidths(fullWidth, maxKeyLen int) (sprintfLen int, valueLen int) {
+	totalWidth := utils.FooterWidth(fullWidth)
+	valueLen = totalWidth - maxKeyLen - 2
+	if valueLen < totalWidth/2 {
+		valueLen = totalWidth/2 - 2
+		sprintfLen = valueLen
+	} else {
+		sprintfLen = maxKeyLen + 1
 	}
-	r.SetBorderInfoItems(fmt.Sprintf("%d/%d", renderIndex, len(m.fileMetaData.metaData)))
+	return
```

golangci-lint (1.64.8)

naked return in func `computeWidths` with 10 lines of code

(nakedret)